### PR TITLE
[ISSUE #1687] Fix the problem that the heartbeat cannot be reported

### LINKED ISSUE #1687: When using the Polaris registry, InstanceRegisterRequest needs to set the value of the TTL field
**What happened**:

If the TTL field is not set, polaris will think that this instance does not need to perform the heartbeat health check operation, then after the instance goes offline, the instance cannot be converted to unhealthy normally.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

### PR DESCRIPTION
**Pull request by chuntaojun**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #1687 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: registry/polaris/registry.go [hunk 1]
```diff
@@ -301,6 +301,10 @@ func createRegisterParam(url *common.URL, serviceName string) *api.InstanceRegis
 			Metadata:  metadata,
 		},
 	}
+
+	req.SetTTL(5)
```
Comment (justxuewei) [inline registry/polaris/registry.go:305]: Why do you use 5 instead of other number here?

Comment (chuntaojun) [reply registry/polaris/registry.go:305]: Nacos default heartbeat reporting time at 5 seconds, so here is a direct reference

Comment (justxuewei) [reply registry/polaris/registry.go:305]: Ok, could you define a constant and replace 5 with it here?

Comment (chuntaojun) [reply registry/polaris/registry.go:305]: ok

**Comment by justxuewei** [comment]
Please perform `go fmt ./...` on your local environment, and push the code again to pass the CI.

**Review by AlexStocks — APPROVED** [review]

**Review by zhaoyunxing92 — APPROVED** [review]

**Comment by liu-song** [comment]
https://github.com/apache/dubbo-go/blob/master/registry/polaris/service_discovery.go#L97，这里log里面的 nacos  可以顺带改下

**Review by AlexStocks — APPROVED** [review]

**Review by LaurenceLiZhixin — APPROVED** [review]

**Pull request merged**